hybrid-iam-ra-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+exec >&2
+set -eux -o pipefail
+
+declare -r USER_DATA="/.bottlerocket/host-containers/current/user-data"
```

Comment:
This doesn't look right.

A bootstrap container would have its user data in `/bottlerocket/bootstrap-containers/current/user-data` - but in this case the user data will be the name of this shell script as the command to execute, in base64 form.

Reply:
Ah yeah you're right. Fixed it.

Reply:
Ah yeah you're right, fixed it